src/directives/rcdni.js

```diff
@@ -0,0 +1,42 @@
+directives.directive('rcDni',['dniSemanticsValidationService','dniSyntaxValidationService',function(dniSemanticsValidationService,dniSyntaxValidationService){
+	return{
+		require: 'ngModel',					
+		link: function (scope, elm, attrs, ctrl){	
+			var validationToExecute = function (viewValue){
+				var currentTypedValue = viewValue;
+				validateDNI(currentTypedValue);
+			};
+
+			function validateDNI(dni){
+				if (validateDNISyntax(dni)){
+					validateDNISemantics(dni);
```

Comment:
Can you talk with Evaristo and ask to implement changes then merge from his branch? let's have the full pie up and running